### PR TITLE
Change default thread count

### PR DESCRIPTION
**Pull request by ronenhamias**

### CODE REVIEW

File: src/main/java/io/servicefabric/socketio/SocketIOServer.java [hunk 1]
```diff
@@ -26,8 +26,11 @@
 import javax.net.ssl.SSLContext;
 
 import io.netty.util.HashedWheelTimer;
+
+import org.omg.CORBA.Environment;
```
Comment (myroslavlisniak) [inline src/main/java/io/servicefabric/socketio/SocketIOServer.java]: I think we don't need CORBA here :)

File: src/main/java/io/servicefabric/socketio/SocketIOServer.java [hunk 2]
```diff
@@ -91,8 +94,9 @@ public synchronized void start() {
                 configuration, listener,
                 sslContext
         );
+        NioEventLoopGroup group = new NioEventLoopGroup(2* Runtime.getRuntime().availableProcessors());
```
Comment (myroslavlisniak) [inline src/main/java/io/servicefabric/socketio/SocketIOServer.java]: you can use constructor without parameters, this is default behaviour

File: src/main/java/io/servicefabric/socketio/SocketIOServer.java (hunk 1 above)
Comment (antonkharenko) [inline src/main/java/io/servicefabric/socketio/SocketIOServer.java]: Good that they already have OMG in their package structure :)

**Comment by antonkharenko** [comment]
This change do not actually make any difference

**Pull request closed without merging**